docs/MakeOTFUserGuide.md

```diff
@@ -38,90 +41,81 @@ MyFontFamily/
-|`-ni`| |Turn off the Italic style. Can be used to override a project file setting, otherwise has no effect.|
 |`-nS`| |Turn off subroutinization.|
 |`-nga`| |Turn off applying the GlyphOrderAndAliasDB file|
-|`-naddn`| |Turn off adding a standard .notdef. Can be used to override a project file setting, otherwise has no effect.|
-|`-nadds`| |Turn off adding Apple symbol glyphs. Can be used to override a project file setting, otherwise has no effect.| 
 
-Options are applied in the order in which they are specified: `–r –nS` will not subroutinize a font, but `–nS –r` will subroutinize a font. Option values are read (in order of increasing priority) from first the fontinfo file keyword/value pairs, then the specified project file, if any, and then from the command line, in order from left to right. 
+Options are applied in the order in which they are specified: `–r –nS` will not subroutinize a font, but `–nS –r` will subroutinize a font. Option values are read (in order of increasing priority) from first the fontinfo file keyword/value pairs, and then from the command line, in order from left to right. 
 
-Subroutinization is a process by which common elements in a set of glyphs are decomposed in separate subroutines. This can reduce the size of the final font but can require extreme amounts of memory and time for a large font, such as CID fonts. MakeOTF may need as much as 64 Mbytes of memory for large Roman fonts, and will do most with only 32 Mbytes, but it may need 768 Mbytes of memory to subroutinize a 5-Mbyte CID font. Subroutinizing is reasonably fast when done all in memory: a few minutes for a Roman font, half an hour to three hours for a CID font. However, if the system must use virtual memory, the time required can increase by a factor of 20 or more. Subroutinizing is likely to be useful for Roman fonts, and for Korean CID fonts. Japanese and Chinese CID fonts usually only yield a few percent size reduction with subroutinizing, due to fewer repeating path elements.
+Subroutinization is a process by which common elements in a set of glyphs are decomposed in separate subroutines. This can reduce the size of the final font but can require extreme amounts of memory and time for a large font, such as CID fonts. `makeotf` may need as much as 64 Mbytes of memory for large Roman fonts, and will do most with only 32 Mbytes, but it may need 768 Mbytes of memory to subroutinize a 5-Mbyte CID font. Subroutinizing is reasonably fast when done all in memory: a few minutes for a Roman font, half an hour to three hours for a CID font. However, if the system must use virtual memory, the time required can increase by a factor of 20 or more. Subroutinizing is likely to be useful for Roman fonts, and for Korean CID fonts. Japanese and Chinese CID fonts usually only yield a few percent size reduction with subroutinizing, due to fewer repeating path elements.
```

Comment:
This can be:

> Subroutinization is a process by which common elements in a set of glyphs are decomposed in separate subroutines, reducing the size of the final font. Subroutinizing should take at most a few minutes for a typical Roman font, but could take up to 3 hours for a large CJK font.
Subroutinizing is likely to be most useful for Roman and Korean fonts. Japanese and Chinese fonts usually only yield a few percent size reduction with subroutinizing, due to their being fewer repeating path elements.

Reply:
I have a feeling that talking about lengths of time won’t age well. I’ve never seen “minutes” needed for Western fonts – usually, the operation is much quicker.